src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -1,36 +1,52 @@
-import { Component } from '@angular/core';
+import { Component, OnDestroy } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { forkJoin, Subscription } from 'rxjs';
+import { finalize } from 'rxjs/operators';
 
 import { ClusterService } from '~/app/shared/api/cluster.service';
-import { AppConstants } from '~/app/shared/constants/app.constants';
+import { HostService } from '~/app/shared/api/host.service';
+import { ActionLabelsI18n, AppConstants } from '~/app/shared/constants/app.constants';
 import { NotificationType } from '~/app/shared/enum/notification-type.enum';
-import { Permission } from '~/app/shared/models/permissions';
+import { Permissions } from '~/app/shared/models/permissions';
+import { WizardStepModel } from '~/app/shared/models/wizard-steps';
 import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
 import { NotificationService } from '~/app/shared/services/notification.service';
+import { WizardStepsService } from '~/app/shared/services/wizard-steps.service';
 
 @Component({
   selector: 'cd-create-cluster',
   templateUrl: './create-cluster.component.html',
   styleUrls: ['./create-cluster.component.scss']
 })
-export class CreateClusterComponent {
-  permission: Permission;
-  orchStatus = false;
-  featureAvailable = false;
+export class CreateClusterComponent implements OnDestroy {
+  currentStep: WizardStepModel;
+  currentStepSub: Subscription;
+  permissions: Permissions;
   projectConstants: typeof AppConstants = AppConstants;
+  hosts: Array<object> = [];
+  stepTitles = ['Add Hosts', 'Review'];
```

Comment:
This line I'm also updating so maybe no need I think..as this PR  just introduces the add-host page `Review` shouldn't be there maybe?